docker/Dockerfile.test.sqlite.clang7.bionic

```diff
@@ -0,0 +1,20 @@
+
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y software-properties-common wget && \
```

Comment:
Note: Always keep as few as layers as you can. This can reduce the image size.

Format the **RUN** command by breaking the lines where it is possible:
```Dockerfile
RUN apt-get update
  && apt-get install -y software-properties-common wget \
  && wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key| apt-key add - \
  && add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" \
  && apt-get update \
  && apt-get install -y clang-7
       clang-tidy-7 \
       doxygen \
       libpq-dev \
       thrift-compiler \
       make \
       build-essential \
       curl \
       doxygen \
       gcc-multilib \
       git \
       python-virtualenv \
       python-dev \
       libsasl2-dev \
       python-dev \
       libldap2-dev \
       libssl-dev \
  && update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-7 100 \
  && update-alternatives --install /usr/bin/clang clang /usr/bin/clang-7 100 \
  && update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-7 100
```